docs/guides/operator-private-gcp-confidential-space.md

```diff
@@ -0,0 +1,405 @@
+  --image-family confidential-space \
+  --service-account {SERVICE_ACCOUNT} \
+  --metadata ^~^tee-image-reference=ghcr.io/iabtechlab/uid2-operator@sha256:{IMAGE_SHA}~tee-restart-policy=Never~tee-env-DEPLOYMENT_ENVIRONMENT=prod~tee-env-API_TOKEN={API_TOKEN}
+```
+
+(**GWH_YS14 URL update please for the above URL**)
+
+(**GWH_YS15 Andrei said: "Where would this be accessible from?" and "Just give full different command lines for prod and integ." please help with that.**)
+
+>NOTE: This script includes an additional option not used in the script for the integration environment, `--machine-type n2d-standard-16`. By specifying this value you can make sure that the production deployment of UID2 Operator runs on the recommended machine type that matches the production configuration.
+
+### Deployment Example Using Terraform for Load Balancing
+
+For ease of deployment and upgrade, you can use a Terraform template to deploy a UID2 Private Operator implementation with load balancing and auto-scaling features. In this scenario, all VM instances run on Confidential Space VMs, and are deployed in multiple AZs. (**GWH_YS16 what is AZ?**)
+
+The Terraform template creates the following components:
+
+- Network: VPC and subnetwork.
+- Instances: Instance template, instance groups (with autoscaling enabled)
+- Ingress: Load balancer (with health check), forwarding rules, firewall rules.
+- Egress: NAT. (**GWH_YS17 what is NAT?**)
+
+In this section:
+- [Install Terraform](#install-terraform)
+- [Set Up the Environment](#set-up-the-environment)
+- [Run Terraform](#run-terraform)
+- [Test](#test)
+- [Clean Up](#clean-up)
+- [Inputs](#inputs)
+- [Outputs](#outputs)
+- [Terraform Template&#8212;Notes](#terraform-templatenotes)
+
+#### Install Terraform
+
+Install Terraform if it is not already installed: visit [terraform.io](https://www.terraform.io/).
+
+#### Set Up the Environment
+
+1. Set the project, replacing the `{PROJECT_ID}` placeholder with your own project ID (see [Confidential Space Account Setup](#confidential-space-account-setup)):
+
+   ```
+   gcloud config set project {PROJECT_ID}
+   ```
+
+2. Configure the environment for Terraform:
+
+   ```
+   gcloud auth application-default login
+   ```
+
+#### Run Terraform
+
+Run the following:
+
+```
+terraform init
+terraform apply
+```
+
+#### Test
+
+1. Get the public IP address for the load balancer:
+
+   ```
+   terraform output load-balancer-ip
+   ```
+
+2. Check health at the health check endpoint (for details, see [Test Using the Health Check Endpoint](#test-using-the-health-check-endpoint)):
+
+   ```
+   http://{PUBLIC_IP}/ops/healthcheck
+   ```
+
+#### Clean Up
+
+Remove all resources created by Terraform:
+
+```
+terraform destroy
+```
+
+#### Inputs
+
+(**GWH_YS18 I need more data re the inputs and outputs so we can have some little intro line to these tables. Is this something they clean up? Or something that's required? Please help with some explanation for both inputs and outputs.**)
```

Comment:
Fields marked as required - no default value, partner needs to provide.
Fields marked as not required - have default value. Customer could also tune to meet their requirements.
- region : they could chose to deploy to any region
- network_name : the vpc resource name ( also used for rules/ instance tags )
- uid_machine_type: we should add a description that do not change this value for prod. In integ they could use whatever machine type they want. 
- min_replicas/max_replicas : the scale, at customer's choice.
- uid_deployment_env: "integ" or "prod", if not specified we will use integ.
- debug_mode: do not set to true unless you are working with uid team to debug issues. Otherwise you can not pass attestation.

Output:
Public ip of load balancer. It could be used by customer to sanity check (health check)/ configure DNS.